### ISSUE TITLE
Crash - Empty texmap texture with index 13629!

### ISSUE THREAD

**Issue by PatrikSamuelTauchim** [ZaneDubya/UltimaXNA#219]
You are welcome to fix this :-)

I haven't experienced crash for a long time.

```
System.Exception was unhandled
  HResult=-2146233088
  Message=Empty texmap texture with index 13629!
  Source=UltimaXNA
  StackTrace:
       at UltimaXNA.Core.Diagnostics.Listeners.MsgBoxOnCriticalListener.OnEventWritten(EventLevels level, String message) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Core\Diagnostics\Listeners\MsgBoxOnCriticalListener.cs:line 21
       at UltimaXNA.Core.Diagnostics.Listeners.AEventListener.OnEventWritten(EventLevels level, String message, Object[] args) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Core\Diagnostics\Listeners\AEventListener.cs:line 22
       at UltimaXNA.Core.Diagnostics.Tracing.Tracer.Critical(String message, Object[] args) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Core\Diagnostics\Tracing\Tracer.cs:line 48
       at UltimaXNA.Ultima.IO.TexmapData.readTexmapTexture(Int32 index) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\IO\TexmapData.cs:line 55
       at UltimaXNA.Ultima.IO.TexmapData.GetTexmapTexture(Int32 index) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\IO\TexmapData.cs:line 37
       at UltimaXNA.Ultima.World.EntityViews.GroundView..ctor(Ground ground) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\World\EntityViews\GroundView.cs:line 36
       at UltimaXNA.Ultima.World.Entities.Ground.CreateView() in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\World\Entities\Ground.cs:line 51
       at UltimaXNA.Ultima.World.Entities.AEntity.GetView() in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\World\Entities\AEntity.cs:line 172
       at UltimaXNA.Ultima.World.Maps.TileSorter.GetSortValues(AEntity e, Int32& z, Int32& threshold, Int32& type, Int32& tiebreaker) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\World\Maps\TileSorter.cs:line 110
       at UltimaXNA.Ultima.World.Maps.TileSorter.Compare(AEntity x, AEntity y) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\World\Maps\TileSorter.cs:line 52
       at UltimaXNA.Ultima.World.Maps.TileSorter.Sort(List`1 items) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\World\Maps\TileSorter.cs:line 28
       at UltimaXNA.Ultima.World.Maps.MapTile.get_Entities() in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\World\Maps\MapTile.cs:line 237
       at UltimaXNA.Ultima.World.WorldViews.MiniMapChunk..ctor(MapChunk block) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\World\WorldViews\MiniMapChunk.cs:line 74
       at UltimaXNA.Ultima.World.WorldViews.MiniMapTexture.InternalQueueMapBlock(Map map, UInt32 cellx, UInt32 celly) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\World\WorldViews\MiniMapTexture.cs:line 186
       at UltimaXNA.Ultima.World.WorldViews.MiniMapTexture.Update(Map map, Position3D center) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\World\WorldViews\MiniMapTexture.cs:line 77
       at UltimaXNA.Ultima.World.WorldView.Draw(Double frameTime) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Ultima\World\WorldView.cs:line 120
       at UltimaXNA.UltimaGame.Draw(GameTime gameTime) in C:\Users\VM\Desktop\UltimaXNA-master\dev\UltimaGame.cs:line 211
       at Microsoft.Xna.Framework.Game.DrawFrame()
       at Microsoft.Xna.Framework.Game.Tick()
       at Microsoft.Xna.Framework.Game.HostIdle(Object sender, EventArgs e)
       at Microsoft.Xna.Framework.GameHost.OnIdle()
       at Microsoft.Xna.Framework.WindowsGameHost.RunOneFrame()
       at Microsoft.Xna.Framework.WindowsGameHost.ApplicationIdle(Object sender, EventArgs e)
       at System.Windows.Forms.Application.ThreadContext.System.Windows.Forms.UnsafeNativeMethods.IMsoComponent.FDoIdle(Int32 grfidlef)
       at System.Windows.Forms.Application.ComponentManager.System.Windows.Forms.UnsafeNativeMethods.IMsoComponentManager.FPushMessageLoop(IntPtr dwComponentID, Int32 reason, Int32 pvLoopData)
       at System.Windows.Forms.Application.ThreadContext.RunMessageLoopInner(Int32 reason, ApplicationContext context)
       at System.Windows.Forms.Application.ThreadContext.RunMessageLoop(Int32 reason, ApplicationContext context)
       at System.Windows.Forms.Application.Run(Form mainForm)
       at Microsoft.Xna.Framework.WindowsGameHost.Run()
       at Microsoft.Xna.Framework.Game.RunGame(Boolean useBlockingRun)
       at Microsoft.Xna.Framework.Game.Run()
       at UltimaXNA.Bootstrapper.StartEngine() in C:\Users\VM\Desktop\UltimaXNA-master\dev\Bootstrapper.cs:line 79
       at UltimaXNA.Bootstrapper.Initialize() in C:\Users\VM\Desktop\UltimaXNA-master\dev\Bootstrapper.cs:line 62
       at UltimaXNA.Bootstrapper.Main(String[] args) in C:\Users\VM\Desktop\UltimaXNA-master\dev\Bootstrapper.cs:line 29
       at System.AppDomain._nExecuteAssembly(RuntimeAssembly assembly, String[] args)
       at System.AppDomain.ExecuteAssembly(String assemblyFile, Evidence assemblySecurity, String[] args)
       at Microsoft.VisualStudio.HostingProcess.HostProc.RunUsersAssembly()
       at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
       at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
       at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
       at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
       at System.Threading.ThreadHelper.ThreadStart()
  InnerException: 
```

**Comment by ZaneDubya** [comment]
Interesting! Where did that happen? (on the map).

**Comment by PatrikSamuelTauchim** [comment]
@ZaneDubya 1445 3745 0 Trammel

**Comment by ZaneDubya** [comment]
Thanks

**Comment by ZaneDubya** [comment]
You know, my muls don't have a land tiledata with index 328. Bizarre.

**Comment by ZaneDubya** [comment]
Fixed in 44a23a7. Closing.
